package.json

```diff
@@ -47,12 +49,14 @@
     "test-proto": "yarn compile-proto && yarn build-proto-roundtrip",
     "unit-core": "yarn jest \"lighthouse-core\"",
     "unit-cli": "yarn jest \"lighthouse-cli/\"",
```

Comment:
this cmd should still work.
i guess you can just put the mocha cmd in here now?